tests/framework/extensions/clusters/dynamicSchema.go

```diff
@@ -17,12 +15,11 @@ type SSHUser struct {
 	Description string `json:"description"`
 }
 
-// ResourceFields contains all the fields of the resources found in DynamicSchemaSpec
 type ResourceFields struct {
 	SSHUser SSHUser
 }
 
-// DynamicSchemaSpec contains ResourceFields that contains all the data for the DynamicSchemaSpec which a type in provisinong.cattle.io.clusters this is how we get an ssh user for a node pool
+// DynamicSchemaSpec contains ResourceFields that contains all the data for the DynamicSchemaSpec which a type in provisioning.cattle.io.clusters this is how we get an ssh user for a node pool
```

Comment:
these changes were made from this request https://github.com/rancher/rancher/pull/42706#discussion_r1329466653